staticfiles/templates/contributors/radhika.html

```diff
@@ -0,0 +1,107 @@
+					  	<p>I love learning and experimenting with new technologies. I also like reading about things I haven't ventured to before! I've recently gained an interest in HCI, and am looking forward to doing some good work in the area! :)</p>
+					  	<!-- <p class="pull-right red"><i class="icon-heart"></i></p> -->
+					  </div><!-- tab about -->
+					  
+	    			  <!-- ===== Second Tab ===== -->
+					  <div class="tab-pane" id="profile">
+					  	<h4><i class="icon-briefcase"></i> Job Experience</h4>
+					  	<p class="sm">
+					  		<a href="inmobi.com"><grey>InMobi</grey></a> | Aug 2014 - Present.<br>
+					  		<!-- <grey>Other Corp.</grey> | January 2011 - May 2013. --><br>
+					  	</p>
+					  
+						  <h4><i class="icon-file-text-alt"></i> Education</h4>
+					  	<p class="sm">
+					  		<grey>Bachelor of Engineering</grey> | PESIT-BSC.<br>
+					  		<grey>High School</grey> | National Public School, Indiranagar.<br>
+					  	</p>
+					  	
+						  <h4><i class="icon-trophy"></i> Awards</h4>
+					  	<p class="sm">
+					  		<grey>Entrepreneural Idea Award</grey> | ICICI Trinity<br>
+					  		<grey>Elite Student Award</grey> | 2012.<br>
+					  	</p>
+					  </div><!-- Tab Profile -->
+					  
+	    			 
+					  
+	    			  <!-- ===== Fourth Tab ===== -->
+					  <div class="tab-pane" id="contact">
+						  <h4>Contact Information</h4>
+						  <div style='width: 100%; height: 2px; background-color: #666;margin-bottom::10px;'></div>
+						  	<br>
+						  <div class="row">
+							  <div class="col-xs-12">
+								  <p class="sm">
+								  	<a style="color:black;"<i class="icon-globe"></i> - radhikabhanu.in </a><br>
+									<i class="icon-envelope"></i> - radhika.bhanu@gmail.com  
+								  </p>
+							  </div><!-- col-xs-6 -->
+							  
+							  <div class="col-xs-12">
+								  <p class="sm">
+								  	<i class="icon-phone"></i> - +91 9916541036 <br>
+									<a style="color:black;"><i class="icon-github"></i> - github.com/radhikabhanu </a>
+								  </p>
+							  </div><!-- col-xs-6 -->
+						  </div><!-- row -->
+						  
+						  <h4>Social Links</h4>
+						  <div style='width: 100%; height: 2px; background-color: #666;margin-bottom::10px;'></div>
+						  	<br>
+						  <div class="row">
+							  <div class="col-xs-12">
+								  <p class="sm">
+								  	<a style="color:black;" href="http://facebook.com/radhika.bhanu"<i class="icon-facebook"></i> - radhika.bhanu </a><br> 
+								  </p>
+							  </div><!-- col-xs-6 -->
+						  </div><!-- row -->
+					  </div><!-- Tab Contact -->
+					  
+					</div><!-- Tab Content -->
+    			</div><!-- col-md-8 -->
+    		</div><!-- row w -->
+    	</div><!-- col-lg-6 -->
+    </div> <!--front -->
+    </div><!-- /.container -->
+
+
+    <!-- Bootstrap core JavaScript
+    ================================================== -->
+    <!-- Placed at the end of the document so the pages load faster -->
+	<script>
+	$('#myTab a').click(function (e) {
```

Comment:
If you'd like to use, that's great but I strongly recommend <b>not</b> using jquery. We use angular-strap so you can use this to create tabs - http://mgcrea.github.io/angular-strap/#/tabs
I recommend defining a controller for your route, in that enable tabs using angular